erpnext/selling/doctype/quotation/quotation.json

```diff
@@ -91,6 +93,7 @@
   "terms",
   "signature_section_section",
   "signature",
```

Comment:
@sahil28297
signature is already there, I have added the customer signature field.
Please check